docs/kratos/guides/tracing.mdx

```diff
@@ -9,15 +9,17 @@ It's yet another tool that you can use to aid you in profiling, debugging and
 ultimately understanding your deployment of Kratos better. Kratos supports the
 following tracing options:
 
-- Tracing backend(s): Jaeger, Elastic APM, Datadog, Zipkin and Instana - _Note:
+- Tracing backend(s): Jaeger - _Note:
   adding support for other
-  [opentracing compliant backends](https://opentracing.io/docs/supported-tracers)
+  [OpenTelemetry exporters](https://opentelemetry.io/docs/instrumentation/go/exporting_data/)
   is planned. To aid in priority, please
   [create an issue](https://github.com/ory/kratos/issues) with your feature
   request._
 - Following existing traces: If you have deployed Kratos behind a proxy that has
   initiated a trace, Kratos will attempt to join that trace by examining the
-  request headers for tracing context.
+  request headers for tracing context. At present, support for Jaeger
+  and Zipkin (B3) spans exists; support for additional formats like Trace Context
+  and Baggage is planned.
 
```

Comment:
Can you add a short section on how to configure tracing here please?

Reply:
Done. Let me know if any more info is needed.